.github/workflows/publish_nuget.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Nuget
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Does Github expose an `on:push:tags` for actions?

Reply:
Yes: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#create

Reply:
Should we use that to create a release instead? Then a tag for `1.2.0` would kick off the release process for that version. I understand wanting more of the "nightly" kind of builds too so pushing to master makes sense, but I think doing the actual release with tags makes more sense.

Reply:
Depends on how the maintainers want to manage this. For me personally, I prefer managing the version number from the `*.fsproj`. Making a tag in git to trigger a package release doesn't suit my style. I'm also unaware of any github action that uploads to Nuget upon tag creation, so we might have to make one for ourselves.

Side note, [the github action being used](https://github.com/marketplace/actions/publish-nuget) automatically creates a tag with the appropriate version number (if the push to Nuget is successful.)

Reply:
We can try it and see how it goes. Nothing stops us from using that other action that gets triggered upon tag creation.